hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/OzoneAddress.java

```diff
@@ -343,6 +350,35 @@ public void ensureKeyAddress() throws OzoneClientException {
     }
   }
 
+  /**
+   * Checking for a volume and a bucket
+   * but also accepting a snapshot
+   * indicator and a snapshot name.
+   * If the keyName can't be considered
+   * a valid snapshot, an exception is thrown.
+   *
+   * @throws OzoneClientException
+   */
+  public void ensureSnapshotAddress()
+      throws OzoneClientException {
+    if (keyName.length() > 0) {
+      if (OmUtils.isBucketSnapshotIndicator(keyName)) {
+        snapshotNameWithIndicator += keyName;
```

Comment:
Is there a reason to use `+=` instead of `=` here?

What if `ensureSnapshotAddress()` is called twice?

Reply:
You are right, I'll fix it!